fetch.bs

```diff
@@ -3237,6 +3566,31 @@ in addition to <a>HTTP fetch</a> above.
   nullity has already been checked. The <a lt=extract for=BodyInit>extracting</a> operation cannot
   throw as it was called for the same <a for=body>source</a> before.
 
+ <li><p>Let <var>useReferredTokenBinding</var> be the result of
+ <a for=header>parsing</a> `<a http-header><code>Include-Referred-Token-Binding-ID</code></a>`
+ in <var>actualResponse</var>'s <a for=response>header list</a>.
+
+ <li><p>Set the <var>request</var>'s <a for=request>referred-token-binding origin</a> to null.
+
+ <li><p>If <var>useReferredTokenBinding</var> is `<code>true</code>`
```

Comment:
The current behavior in Chromium is to let headers coalesce, and then compare to the string "true", meaning that if the header is sent multiple times we treat it as if the header was set to "false" (or any other value, or not present). I'm not sure that's the behavior we actually want.

If we want to reject the response (instead of ignoring) if the header has a bad value, another thing to consider is whether the header was sent but Token Binding wasn't negotiated for the connection (or a Token Binding wasn't sent on the request). Right now Chromium will ignore the header if Token Binding wasn't negotiated.

Reply:
Per the TOKBIND-HTTPS spec, this header has a single valid value (not a list), so servers must not emit multiple headers (per HTTP spec).

As currently described, multiple headers would be treated as an invalid value.
Reasonable?

@nharper , here useReferredTokenBinding causes the referred-token-binding origin to be set for the upcoming redirect request only if the request is currently using token binding.
Need more clarification?